src/shared/components/oncoprint/controls/ConfirmNgchmModal.tsx

```diff
@@ -26,21 +26,14 @@ export default class ConfirmNgchmModal extends React.Component<{ show:boolean, o
                 <Modal.Body>
                 	<div className="oncoprint__controls__heatmap_menu mdacc-modal">
                     <p>Continue will open a tab or window to a different site (not cBioPortal).</p>
-                    <img src={require("./mdandersonlogo260x85.png")} alt="MD Anderson Cancer Center logo" style={{width:153, height:50, margin:2, 'margin-right':20, 'margin-bottom':20, float:'left'}}/>
                     <p>
-                         The University of Texas MD Anderson Cancer Center has created a 
-                        Next-Generation Clustered Heatmap compendium
-                        based on the same dataset or a very similar dataset containing many of the
-                        same TCGA samples.  Continue will display a page of NG-CHM based on 
-                        this dataset.
+                    You will have the option to open the Next-Generation Clustered 
+                    Heatmaps (NG-CHM) developed by The University of Texas 
+                    MD Anderson Cancer Center for the same selected study.
+                    </p>
+                    <p className="styleMyItalicText">
```

Comment:
Just surround text in `<em>` to avoid this custom class

Reply:
Ok.  Following your suggestion, which means the css class can go away.  (I just normally put all formatting in CSS.)